samples/openthread/cli/prj.conf

```diff
@@ -15,3 +15,14 @@ CONFIG_SHELL_CMD_BUFF_SIZE=416
 
 # Enable OpenThread features set
 CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER=y
+
+CONFIG_NET_L2_OPENTHREAD=y
+
+# Generic networking options
+CONFIG_NETWORKING=y
+CONFIG_NET_CONNECTION_MANAGER=y
```

Comment:
`CONFIG_NET_CONNECTION_MANAGER` can be removed from all of our samples since we are not using it.